infrastructure-playbooks/filestore-to-bluestore.yml

```diff
@@ -39,17 +39,22 @@
       block:
         - import_role:
             name: ceph-facts
+            tasks_from: container_binary.yml
+
+        - name: set_fact container_run_cmd
+          set_fact:
+            container_run_cmd: "{{ container_binary + ' run --rm --privileged=true --net=host --pid=host --ipc=host -v /dev:/dev -v /etc/ceph:/etc/ceph -v /var/lib/ceph:/var/lib/ceph -v /var/run:/var/run --entrypoint=' if containerized_deployment else '' }}ceph-volume {{ ceph_docker_registry + '/' + ceph_docker_image + ':' + ceph_docker_image_tag if containerized_deployment else '' }}"
+
+        - name: set_fact container_exec_cmd
+          set_fact:
+            container_exec_cmd: "{{ container_binary + ' exec ceph-mon-' + hostvars[groups[mon_group_name][0]]['ansible_hostname'] if containerized_deployment else '' }}"
```

Comment:
Could we merge both set_fact in a single task ?

Also there's a missing `| bool` filter on the `containerized_deployment` variable